ui-tests/src/test/java/io/syndesis/qe/pages/integrations/summary/Details.java

```diff
@@ -153,8 +152,8 @@ public SelenideElement getPublishedVersion() {
     }
 
     public String getApiUrl() {
-        return $(Element.INTEGRATION_DESCRIPTION)
-            .$$("input").shouldHaveSize(1).first().shouldBe(enabled, visible)
+        return getRootElement()
+            .findAll("input").shouldHaveSize(1).first().shouldBe(enabled, visible)
```

Comment:
since we are touching this code, could we change the `findAll("input").shouldHaveSize(1)` to `$("input")`?

Reply:
What if he wants to check that there is just 1 input element on the page?

Reply:
I don't know whether the author wants to be sure that there is just 1 input element or not. So I didn't refactor the logic, only I changed the root element.
Btw, is any differences between `element.find()` or `element.$()` ? I considered it equivalent and find () is more readable for me.

Reply:
Ok, let's leave it as it is. And you're right, those are synonyms, I guess the `$` comes from the jQuery tradition and feels more idiomatic.